### PR TITLE
BXMSDOC-3331 added ddl script steps

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira:
[BXMSDOC-3331](https://issues.jboss.org/browse/BXMSDOC-3331)

### CODE REVIEW

File: doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc [hunk 1]
```diff
@@ -22,7 +22,18 @@ endif::[]
 ** Sybase: `org.hibernate.dialect.SybaseASE157Dialect`
 
 .Procedure
-. Unzip the `rhpam-7.4.0-maven-repository.zip` offline Maven repository file.
+. Prepare your database:
+.. Unzip `{PRODUCT_INIT}-{PRODUCT_VERSION}.x-add-ons.zip` in a temporary directory, for example `TEMP_DIR`.
+.. Unzip `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool.zip`.
+.. Change directory to the `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool/ddl-scripts` directory. This directory contains DDL scripts for several database types.
```
Comment (pnagesha) [inline doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc]: What are you changing? Are you moving files to the new directory or are you navigating to the other directory to do something?

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc [hunk 2]
```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
 * The JDBC drivers that you want to use to create database connections are installed on all servers on which you want to deploy the data source. Some JDBC drivers are installed with {WEBLOGIC}, such as WebLogic-branded Data Direct JDBC drivers for DB2, Informix, MS SQL Server, and Sybase. For more information about JDBC drivers, see https://docs.oracle.com/middleware/12213/wls/JDBCA/third_party_drivers.htm#JDBCA231[Using JDBC Drivers with WebLogic Server] in the Oracle Help Center.
 
 .Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
```
Comment (pnagesha) [inline doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc:10]: Go to the <downloads> page. See pg 347 in the style guide.

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc [hunk 3]
```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
...
+* *Product:* {PRODUCT_SHORT}
+* *Version:* {PRODUCT_VERSION}
+. Download *{PRODUCT} {PRODUCT_VERSION}.x Add-Ons*, where *x* is the current patch version.
+. Prepare your database:
```
Comment (pnagesha) [inline doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc]: How are you preparing it? I think the preparing is not step one, but a lead-in sentence.

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc [hunk 4]
```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
...
+.. Unzip `{PRODUCT_INIT}-{PRODUCT_VERSION}.x-add-ons.zip` in a temporary directory, for example `TEMP_DIR`.
+.. Unzip `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool.zip`.
+.. Change directory to the `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool/ddl-scripts` directory. This directory contains DDL scripts for several database types.
```
Comment (pnagesha) [inline doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc]: It is really unclear what the user is expected to do. Are you just asking them to go to a different directory to fetch the scripts?

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc [hunk 5]
```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
...
+.. Import the DDL script for your database into the database that you want to use, for example:
```
Comment (pnagesha) [inline doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc]: It makes it seem like there are 2 different databases.

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc [hunk 6]
```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
...
++
+[source,shell]
+----
+psql jbpm < /ddl-scripts/postgresql/postgresql-jbpm-schema.sql
+----
+
 . In the WebLogic Administration Console, navigate to *Change Center* -> *Lock & Edit*.
```
Comment (pnagesha) [inline doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc:25]: Why are we using "navigate" so much? 
1) It is an awkward word. You are not asking them to muddle through rush hour.
2) The style guide clearly states that use a more direct verb unless you are providing specific directions to a location.

File: doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc [hunk 7]
```diff
@@ -9,7 +9,7 @@ endif::[]
 ifeval::["{context}" == "install-on-tomcat"]
 * {PRODUCT} is installed on {TOMCAT}.
 endif::[]
-* The Red Hat Process Automation Manager 7.4.0 Maven Repository (`rhpam-7.4.0-maven-repository.zip`) file has been downloaded, as described in  <<install-download-proc_{context}>>.
+* The {PRODUCT} {PRODUCT_VERSION}.x Maven Repository (`{PRODUCT_INIT}-{PRODUCT_VERSION}.x-maven-repository.zip`) and the {PRODUCT} {PRODUCT_VERSION}.x Add-Ons (`{PRODUCT_INIT}-{PRODUCT_VERSION}.x-add-ons.zip`) files have been downloaded, as described in  <<install-download-proc_{context}>>. In these files, `x` is the latest patch release.
 * You want to configure one of the following supported databases and Hibernate dialects:
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc:13]: This a prereq so we don't use the imperative.

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc (hunk 2 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc:10]: Good point, but this is the format that we use throughout our docs when referring to the CP. I'll take it up with Stetson (his formulation :-))

File: doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc (hunk 6 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc:25]: I'll talk to Stetson. We use it consistently.

File: doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc [hunk 8]
```diff
@@ -22,7 +22,18 @@ endif::[]
 ** Sybase: `org.hibernate.dialect.SybaseASE157Dialect`
 
 .Procedure
-. Unzip the `rhpam-7.4.0-maven-repository.zip` offline Maven repository file.
+. Prepare your database:
+.. Unzip `{PRODUCT_INIT}-{PRODUCT_VERSION}.x-add-ons.zip` in a temporary directory, for example `TEMP_DIR`.
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc]: Changed in these sections, but changing throughout the repository will be a larger effort.

**Pull request merged**